extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -73,7 +68,52 @@ const LightboxControlsModes = {
   CONTROLS_HIDDEN: 0,
 };
 
-const SWIPE_TO_CLOSE_THRESHOLD = 10;
+/**
```

Comment:
/cc @andrewwatterson to approve these numbers, since they're mostly magic numbers to me that we arrived based on some kind of tweaking. If possible, it would be nice to document some kind of reasoning behind these numbers in case we ever need to change them.

Reply:
Yes, if I had yet more revisions to this doc I was going to take a shot at articulating exactly how we wanted to make this animation feel and why. But this is a fairly complex interaction (as is evidenced by all @sparhami 's work here), and at the end of the day it just has to feel good in a variety of different situations (far image, close image, quick flick, slower drag, et cetera).

If I were a more experienced motion designer I might have a greater ability to think about how the physics should actually act here, but I am not that guy. I'm open to tweaking these numbers in the future, but I think we're well beyond diminishing returns at this point. :)